saleor/plugins/admin_email/emails.py

```diff
@@ -0,0 +1,25 @@
+from .tasks import (
+    send_email_with_link_to_download_file,
+    send_export_failed_info,
+    send_set_staff_password_email_task,
+)
+
+
+def handle_account_set_staff_password(payload):
```

Comment:
I would suggest a different naming here, because if you read this function name, "handle" doesn't say what the function does. Couldn't we just have `send_set_staff_password_email`? The word "account" can be omitted, I think it's obvious.